.github/workflows/consumer_test.yml

```
@@ -10,8 +10,29 @@ env:
   TEST_FOLDER: primer-brand
 
 jobs:
+  changes:
+    runs-on: ubuntu-latest
+    # Required permissions
+    permissions:
+      pull-requests: read
+    # Set job outputs to values from filter step
+    outputs:
+      outputEffected: ${{ steps.filter.outputs.outputEffected }}
```

Nit, but should this be `outputAffected` to use past participle instead of effect? 

```suggestion
      outputAffected: ${{ steps.filter.outputs.outputAffected }}
```

---

Good catch, thanks